common/lib/xmodule/xmodule/tests/test_video.py

```diff
@@ -646,14 +634,70 @@ def test_get_no_video_exists(self, cdn_response):
         self.assertIsNone(get_video_from_cdn(fake_cdn_url, original_video_url))
 
 
-@override_settings(CONTENTSTORE=TEST_DATA_CONTENTSTORE)
-@override_settings(YOUTUBE=TEST_YOU_TUBE_SETTINGS)
 class VideoDescriptorIndexingTestCase(unittest.TestCase):
     """
     Make sure that VideoDescriptor can format data for indexing as expected.
     """
+    def setUp(self):
+        """
+        Overrides YOUTUBE and CONTENTSTORE settings
+        """
+        self.youtube_setting = getattr(settings, "YOUTUBE", None)
+        self.contentstore_setting = getattr(settings, "CONTENTSTORE", None)
+        settings.YOUTUBE = {
+            # YouTube JavaScript API
+            'API': 'www.youtube.com/iframe_api',
+
+            # URL to test YouTube availability
+            'TEST_URL': 'gdata.youtube.com/feeds/api/videos/',
+
+            # Current youtube api for requesting transcripts.
+            # For example: http://video.google.com/timedtext?lang=en&v=j_jEn79vS3g.
+            'TEXT_API': {
+                'url': 'video.google.com/timedtext',
+                'params': {
+                    'lang': 'en',
+                    'v': 'set_youtube_id_of_11_symbols_here',
+                },
+            },
+        }
+
+        settings.CONTENTSTORE = {
+            'ENGINE': 'xmodule.contentstore.mongo.MongoContentStore',
+            'DOC_STORE_CONFIG': {
+                'host': 'localhost',
+                'db': 'test_xcontent_%s' % uuid4().hex,
+            },
+            # allow for additional options that can be keyed on a name, e.g. 'trashcan'
+            'ADDITIONAL_OPTIONS': {
+                'trashcan': {
+                    'bucket': 'trash_fs'
+                }
+            }
+        }
+
+    def tearDown(self):
+        """
+        Returns YOUTUBE and CONTENTSTORE settings to a default value
+        """
+        if self.youtube_setting:
+            settings.YOUTUBE = self.youtube_setting
+            self.youtube_setting.dispose()
```

Comment:
I ran the tests and they pass. I usually use 
`nosetests common/lib/xmodule/xmodule/tests/test_video.py`
just to run that one test, but even after I ran
`paver test` I didn't see any failures.